src/containers/StructurePage/components/SettingsMenu.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Button } from 'ndla-ui';
+import { Settings } from 'ndla-icons/editor';
+import BEMHelper from 'react-bem-helper';
+import SettingsMenuDropdown from './SettingsMenuDropdown';
+
+const classes = new BEMHelper({
+  name: 'settingsMenu',
+  prefix: 'c-',
+});
+
+const SettingsMenu = ({ toggleState, id, toggles, ...rest }) => {
+  const toggleId = `settings-${id}`;
+  return (
+    <div {...classes('')}>
```

Comment:
Did it because I felt the need to control the toggles from the top, to prevent multiple open settings, to be able to close them after network events etc. But I can probably ignore all those reasons if I add the backdrop